<ui version="4.0" >
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Port Setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>330</y>
     <width>501</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>11</y>
     <width>500</width>
     <height>309</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6" >
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" >
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Sensors</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Sensor 1</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="m_sensorCombo1" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Sensor 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="m_sensorCombo2" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>201</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Sensor 3</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="m_sensorCombo3" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Sensor 4</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QComboBox" name="m_sensorCombo4" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Drive</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" >
           <item>
            <widget class="QRadioButton" name="m_singleButton" >
             <property name="text" >
              <string>&amp;Single</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>28</width>
               <height>21</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="m_diffButton" >
             <property name="text" >
              <string>&amp;Differential</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" >
           <item>
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Drive</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_driveCombo" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="insertPolicy" >
              <enum>QComboBox::InsertAlphabetically</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="m_steerLayout" >
           <item>
            <widget class="QLabel" name="m_steerLabel" >
             <property name="text" >
              <string>Steer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_steerCombo" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>152</width>
               <height>0</height>
              </size>
             </property>
             <property name="insertPolicy" >
              <enum>QComboBox::InsertAlphabetically</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" >
           <item>
            <widget class="QCheckBox" name="m_driveCoastCheck" >
             <property name="text" >
              <string>Coast</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>5</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="m_driveRevCheck" >
             <property name="text" >
              <string>Reverse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>Action</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" >
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" >
           <item>
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>Motor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_actionEdit" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" >
           <item>
            <widget class="QCheckBox" name="m_actionCoastCheck" >
             <property name="text" >
              <string>Coast</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>48</width>
               <height>53</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="m_actionRevCheck" >
             <property name="text" >
              <string>Reverse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_sensorCombo1</tabstop>
  <tabstop>m_sensorCombo2</tabstop>
  <tabstop>m_sensorCombo3</tabstop>
  <tabstop>m_sensorCombo4</tabstop>
  <tabstop>m_singleButton</tabstop>
  <tabstop>m_diffButton</tabstop>
  <tabstop>m_driveCombo</tabstop>
  <tabstop>m_steerCombo</tabstop>
  <tabstop>m_driveCoastCheck</tabstop>
  <tabstop>m_driveRevCheck</tabstop>
  <tabstop>m_actionCoastCheck</tabstop>
  <tabstop>m_actionRevCheck</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>m_actionEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>249</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>317</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
